vacuum-advisor/getting-started.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Getting started with pganalyze VACUUM Advisor'
+backlink_href: /docs/vacuum-advisor
+backlink_title: 'VACUUM Advisor'
+---
+
+The VACUUM process is critical because it removes dead rows or unused data from tables and indexes, freeing up space for new data and improving query performance. Without VACUUM, the database is prone to suffer from database bloat, which can impact query performance and reduce system stability over time. By regularly running VACUUM operations, you can prevent database bloat, optimize query performance, and ensure the long-term stability of your database system.
+
+The pganalyze VACUUM Advisor is a tool to observe VACUUM activity and efficiency, and suggests insights that tailor your database’s Postgres configuration to your workloads. VACUUM Advisor uses information pganalyze already collects assuming you are using the latest version of the collector.
+
+## What you get with VACUUM Advisor
+
+- Automatically scan your VACUUM statistics for inefficiencies, and provide insights for how to optimize your Postgres configuration.
+- Find detailed insights to optimize Bloat, Freezing, and Performance issues.
+- Review a prioritized list of insights, so you can quickly work through them, and understand when pganalyze has confirmed there is no action needed.
+- Understand how pganalyze arrives at each insight, so you can make the final decision if and how you want to apply it.
+- Share in-depth autovacuum data with the whole team, and give the data platform team, infrastructure engineers, and DBAs quick access to detailed information.
+- See statistics about VACUUM activity and individual table vacuum activities over time
+
+To get started, you first need to onboard a new database server if you have not yet.
+
+Next, choose VACUUM Advisor in the navigation to view Bloat, Freezing and Performance metrics and insights. For new servers VACUUM metrics will be available after about 10 minutes after the first snapshot was sent by the collector, insights will show up around 24h after onboarding.
+
+## Collector requirements
+
+We recommend installing the latest version of the collector, as we now track more information about your workloads, including table bloat, TOAST table name, reltuples and relpages. At the very least collector [version 0.50.1](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0501------2023-06-29) is required.
```

Comment:
Uhmm "At the very least collector version 0.50.1 is required" sounds too bold, you can get an okay result without having 0.50.1
How about putting this some similar way as Index Advisor, like:

```suggestion
We recommend installing the latest version of the collector, as we now track more information about your workloads, including table bloat, TOAST table name, reltuples and relpages. For best results, make sure you are running collector [version 0.50.1](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0501------2023-06-29) or later.
```